CanvasColorSpaceProposal.md

```diff
@@ -198,9 +198,134 @@ The ``getImageData`` method is responsible for converting the data from the canv
 ## Examples
 
 ### Selecting the best color space match for the user agent's display device
-<pre>
-var colorSpace = window.matchMedia("(color-gamut: p3)").matches ? "display-p3" : "srgb";
-</pre>
+
+This example selects a wide color gamut canvas only if the underlying display has a P3 gamut or larger.
+
+```html
+  // Note that the gamut is named p3, but the color space is 'display-p3'.
+  var matchingColorSpace = window.matchMedia(
+      '(color-gamut: p3)').matches ? 'display-p3' : 'srgb';
+  var canvas = document.getElementById('Canvas');
+  var context = canvas.getContext('2d', {colorSpace:matchingColorSpace});
+```
+
+### Drawing wide color gamut content to a 2D Canvas
+
+This example shows drawing a wide color gamut image to an sRGB and a Display-P3 canvas, and discusses the different expected results.
+
+```html
+  // Let |myWCGImage| be a loaded wide color gamut Image.
+  var myWCGImage;
+
+  // Let |defaultCanvas| be a default (sRGB) canvas. This code will draw the
+  // specified image, but will clip it to the sRGB color gamut.
+  var defaultCanvas = document.getElementById('DefaultCanvas');
+  var defaultCtx = defaultCanvas.getContext('2d');
+  defaultCtx.drawImage(myWCGImage, 0, 0, image.width, image.height)
+
+  // Let |wcgCanvas| be a Display-P3 canvas. This code will draw the specified
+  // image, which will not be clipped to the sRGB color gamut (it will be
+  // restricted only to the P3 color gamut).
+  var wcgCanvas = document.getElementById('DefaultCanvas');
```

Comment:
Not sure this one should be called "DefaultCanvas". Confusing.

Reply:
It's an error -- it should read 'WcgCanvas' (in contrast with the default canvas settings).

Reply:
Sounds fine - looks good now.